### ISSUE TITLE
textarea form label error

### ISSUE THREAD

**Issue by openmindculture** [openmindculture/ingo-steinke.de#238]
supposedly missing label for textarea according to wave audit; false positive as it exists and axe reports zero errors?

![wave form error screenshot](https://github.com/openmindculture/ingo-steinke.de/assets/6377941/953e2163-94f1-4b47-85be-c48e55c3c643)

![axe no errors screenshot](https://github.com/openmindculture/ingo-steinke.de/assets/6377941/2bd2b3d4-5cad-4f34-ab99-dbc0a40045ea)
